.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -69,11 +35,3 @@ export function Playground() {
 * [ ] Updated the component's `README.md` with documentation changes
 * [ ] [Tophatted documentation](https://github.com/Shopify/polaris-react/blob/master/documentation/Tophatting%20documentation.md) changes in the style guide
 * [ ] For visual design changes, pinged one of @ HYPD, @ mirualves, @ sarahill, or @ ry5n to update the Polaris UI kit
```

Comment:
```suggestion
* [ ] For visual design changes, pinged one of @ HYPD, @ mirualves, @ sarahill, or @ ry5n to update the Polaris UI kit
```

Reply:
Yeah that makes sense, let's wait a little bit and see what happens